src/store.js

```diff
@@ -96,6 +106,9 @@ const store = new Vuex.Store({
         context.commit('setJobs', jobs);
       });
     },
+    getGroups(context) {
```

Comment:
The update button of the navbar, should update the groups also, it is necessary to check which is the current component and link dynamically the right method to updated  the elements of the current table.